pkgs/development/python-modules/unrpa/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, fetchPypi, uncompyle6, isPy27 }:
+
+buildPythonPackage rec {
+  pname = "unrpa";
+  version = "2.3.0";
+
+  disabled = isPy27;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0yl4qdwp3in170ks98qnldqz3r2iyzil5g1775ccg98qkh95s724";
+  };
+
+  propagatedBuildInputs = [ uncompyle6 ];
+
+  meta = with lib; {
+    homepage = https://github.com/Lattyware/unrpa;
```

Comment:
rfc 45
```suggestion
    homepage = "https://github.com/Lattyware/unrpa";
```

Reply:
The Python example in the Nixpkgs manual uses the old notation, fixing this.